sql-plugin/src/main/scala/com/nvidia/spark/rapids/RapidsConf.scala

```diff
@@ -999,6 +999,13 @@ object RapidsConf {
     .stringConf
     .createWithDefault(null)
 
+  val SHUFFLE_UCX_MGMT_CONNECTION_TIMEOUT =
+    conf("spark.rapids.shuffle.ucx.managementConnectionTimeout")
+    .doc("The timeout for client connections to a remote peer")
+    .internal()
+    .integerConf
+    .createWithDefault(0)
```

Comment:
Why are we disabling a timeout by default?  Given this is an internal config, curious what the use-case is for this code.  Are we going to suggest users set this undocumented config in certain situations, use it in a test, or ...?

Reply:
It's just for our internal testing, and I expect it to be removed in 21.08. I didn't change it from what it was default (`new Socket(host, port)` does the same). But I can choose a more sensible value.

Reply:
I'd like to be able to control this while we get the UcpListener to be in a state where it can be enabled by default. Then the UCX timeouts will be the ones we use, so this setting won't apply (i.e. it would be via `UCX_` env variables instead).

Reply:
If we're going to rip this out shortly in 21.08 I'm fine with leaving it at what it effectively was before.